core/src/main/java/jenkins/management/SystemInfoLink.java

```diff
@@ -52,4 +56,9 @@ public String getDescription() {
     public String getUrlName() {
         return "systemInfo";
     }
+
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.MANAGE;
```

Comment:
depending on how Jenkins is run - could this expose passwords for TLS certs which are not configurable in the UI?

Reply:
Can you provide an example of what you are thinking about?